vassal-doc/src/main/readme-referencemanual/ReferenceManual/ExpressionProperty.adoc

```diff
@@ -0,0 +1,154 @@
+
+[example]
+GetProperty("Table" + x + "-" + y)
+
+If x currently has a value of 2 and y has a value of 6, then this will return the value of the property _'Table2-6'_.
+
+|===
+
+[width=100%,cols="33%a,67%a"]
+|===
+|*GetString* |Returns the value of the specified property as a String.  +
+
+
+|GetString(prop) | This function returns the value of the property as a String, even if it is a Number.
+
+
+[example]
+GetString("X") +
+
+Return the value of the property named _'X'_ as a String.
+
+|===
+
+[#Map]
+==== Map-based Functions
+These functions return the value of a Map-level property defined on another map
+
+[width=100%,cols="33%a,67%a"]
+|===
+|*GetMapProperty* |Return the value of a property from a different Map .  +
+
+
+|GetMapProperty(prop, map) | Return the value of the property that resides on the named Map.
+
+[example]
+GetMapProperty("SupplyLevel", "SupplyMap") +
+
+Return the value of the Global Property named _'SupplyLevel'_ that is defined on the Map named _'SupplyMap'_.
+
+|===
+
+[#Zone]
+==== Zone-based Functions
+These functions return the value of a Zone-level property defined in another Zone.
+
+[width=100%,cols="33%a,67%a"]
+|===
+|*GetZoneProperty* |Return the value of a property from a different Zone.  +
+
+|GetZoneProperty(prop, zone) +
+
+#[ Trait Only ]#| Return the value of the property that resides in a different named Zone, but on the same Map as the source piece.
+
+[example]
+GetZoneProperty("Supplies", "zone2") +
+
+Return the value of the Property named _'Supplies'_ that is defined in the Zone named _'zone2'_ on the current Map name.
+
+|GetZoneProperty(prop, zone, map) | Return the value of the property that resides in a named Zone on the named Map.
+
+[example]
+GetZoneProperty("Supplies", "zone2", "East") +
+
+Return the value of the Property named _'Supplies'_ that is defined in the Zone named _'zone2'_ on the Map named _'East'_.
+
+|===
+
+[#Attachment]
+==== Attachment-based Functions
+These functions return the value of a Property from a piece attached to the current piece using the <<Attachment.adoc#top, Attachment>> trait.
+
+All Attachment-based Functions will return the value of the property from the first attached piece it that meets the criteria of the function.
+
+Note that if your Attachment includes the 'Allow attachment to self' option, then it is possible for these functions to find and return the Property value from itself.
+
+[width=100%,cols="33%a,67%a"]
+|===
+|*GetAttachProperty* |Return the value of a property from a piece attached to the current piece.  +
+
+|GetAttachProperty(attach, prop)
+
+#[ Trait Only ]#| Return the value of a property _'prop'_ from the first piece attached to the current piece using the attachment named _'attach'_.
+
+[example]
+GetAttachProperty("Supplies", "SUP") +
+
+Return the value of the Property named _'Supplies'_ from the first piece attached to the current piece using the attachment name _'SUP'_.
+
```

Comment:
The example appears to have the arguments in the reverse order to the instructions (e.g., `GetAttachProperty(prop, attach)`.

Reply:
You only fixed the first example; all the later examples have the same issue.